### ISSUE TITLE
TypeScript typing issue with react-redux Provider

### ISSUE THREAD

**Issue by fo-fo** [ctrlplusb/easy-peasy#315]
I get the following typing error with TypeScript (3.3.3, but seems to happen in other versions too) when passing the Easy Peasy (3.0.2) store to react-redux's Provider (`<Provider store={store}>`):

> Property '[Symbol.observable]' is missing in type '{ getState: () => { test: { count: number; dog: { message: string; status: string; }; names: string[]; }; }; subscribe: (listener: () => void) => Unsubscribe; replaceReducer: (nextReducer: Reducer<{ test: { count: number; dog: { message: string; status: string; }; names: string[]; }; }, AnyAction>) => void; ... 7 mo...' but required in type 'Store<any, AnyAction>'.ts(2741)

I found this in the Store interface of redux:
```  /**
   * Interoperability point for observable/reactive libraries.
   * @returns {observable} A minimal observable of state changes.
   * For more information, see the observable proposal:
   * https://github.com/tc39/proposal-observable
   */
  [Symbol.observable](): Observable<S>
```

react-redux version is 7.1.1.

**Comment by mesvil7** [comment]
same issue when update from 2.6.4 to 3.1.0

**Comment by shancial** [comment]
I'm having the same issue, easy-peasy 3.1.0, react-redux 7.1.1 and typings upgraded to latest

**Comment by ctrlplusb** [comment]
Fixed in 3.1.1
